phy/cluster/algorithms.py

```diff
@@ -155,7 +164,7 @@ def __init__(self, tempdir=None, probe=None, **kwargs):
     def _create_filter(self):
         rate = self._kwargs['sample_rate']
         low = self._kwargs['filter_low']
```

Comment:
Uhhhh.

Un-sidetracking this discussion a bit...

Nick, should both these be frequencies in Hz?

This is my main objection before merging!